README.md

```diff
@@ -69,4 +69,21 @@ collecting ...
 
 Results (0.34s):
        2 passed
-```
\ No newline at end of file
+```
+
+## Running web scrapers
+
```

Comment:
Looks like a bad rebase... this was already on `master`

Reply:
Whoops, forgot to change the PR base branch.